enclave_manager/Dockerfile

```diff
@@ -171,19 +171,22 @@ RUN mkdir -p /opt/intel \
  && mkdir -p /etc/init
 WORKDIR /opt/intel
 
-# Install Intel SGX common library
-RUN wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb \
- && dpkg -i libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-enclave-common/libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-epid/libsgx-epid_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-launch/libsgx-launch_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-urts/libsgx-urts_2.10.100.2-bionic1_amd64.deb \
+ && dpkg -i * \
+ && rm -f /usr/lib/x86_64-linux-gnu/libsgx_urts.so
```

Comment:
Can you add a comment why we need to remove libsgx_urts.so

Reply:
It was missed to remove from the Dockerfile. I have removed the line and updated PR